src/components/picture/index.test.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from "@testing-library/react";
+
+import Picture from ".";
+import Source from "./_children/source";
+
+describe("Picture component", () => {
+	it("render a picture element with a nested source element", () => {
+		const { container } = render(
+			<Picture>
+				<Source src="test.jpeg" media="(max-width: 799px)" />
+			</Picture>
+		);
+		expect(container.firstChild).toMatchInlineSnapshot(`
+		<picture
+		  class="c-picture"
+		>
+		  <source
+		    media="(max-width: 799px)"
+		    srcset="test.jpeg"
+		  />
+		</picture>
+	`);
+	});
+
+	it("should render additional classes", () => {
+		const ORIGINAL_CLASSES = "c-picture";
+		const ADDITIONAL_CLASSES = "additionalClass1 additionalClass2";
+		render(
+			<Picture className={ADDITIONAL_CLASSES} data-testid="test-picture">
+				<Source src="test.jpeg" media="(max-width: 799px)" />
```

Comment:
You should never consume the children components directly. This should be consumed via Picture.Source

Reply:
Thanks! Updated.